client/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -29,6 +30,16 @@ export default function Dashboard(): JSX.Element {
 
   return (
     <Grid container component="main" className={`${classes.root} ${classes.dashboard}`}>
+      <Navbar
+        logOut="me"
+        profile="hi"
+        mySitter="My Sitters"
+        becomeSitter="BECOME A SITTER"
+        messages="Messages"
+        loggedInUser={loggedInUser}
```

Comment:
Since `loggedInUser` is provided with a context, there is probably no need to pass this down. We can use the `useAuth` hook as needed within `Navbar` or its sub-components.

Reply:
used useAuth in GetMenuButton component and removed loggedInUser